react/react2/week1/app/src/components/UserSearch.jsx

```diff
@@ -0,0 +1,34 @@
+import React, { useContext } from "react";
+import { GithubContext } from "../GithubContext";
+
+function UserSearch() {
+  const context = useContext(GithubContext);
+  const { query, users, loading, getInput } = context;
+
+  return (
+    <div>
+      <input
+        type="text"
+        value={query}
+        placeholder="insert user"
+        onChange={getInput}
```

Comment:
You are taking event (e) as an argument to getInput(), you should be passing it in here. though it works sometimes, we should follow some rules while coding.